src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts

```diff
@@ -289,6 +319,30 @@ export class SettingsTreeIndicatorsLabel implements IDisposable {
 		}
 	}
 
+	updatePreviewIndicator(element: SettingsTreeSettingElement) {
+		const isPreviewSetting = element.tags?.has('preview');
+		// Hide experimental indicators for now until further review and experimental -> onExP migration.
+		const isExperimentalSetting = false; // element.tags?.has('experimental');
+		this.previewIndicator.element.style.display = (isPreviewSetting || isExperimentalSetting) ? 'inline' : 'none';
+		this.previewIndicator.label.text = isPreviewSetting ?
+			localize('previewLabel', "Preview") :
+			localize('experimentalLabel', "Experimental");
+
+		const content = isPreviewSetting ?
+			localize('previewSettingDescription', "The setting is in the process of becoming stable.") :
+			localize('experimentalSettingDescription', "The setting is experimental and may change names or be removed.");
```

Comment:
Suggested text:

"This setting controls a feature which may change or be removed in the future. Adjusting this setting may result in instability."

"This setting controls a feature which is in active development."